common/templates/_deployment.yaml

```diff
@@ -0,0 +1,141 @@
+{{- define "common.deployment.tpl" -}}
+{{ include "common.deployment.head" . }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        draft: {{ default "draft-app" .Values.draft }}
+        app: {{ template "fullname" . }}
+        release: {{ .Release.Name }}
+{{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+{{- if include "common.registry-pull-secrets" . }}
+      imagePullSecrets:
+      {{- include "common.registry-pull-secrets" . | indent 6 }}
+{{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
@almerico I think it would make sense to add support for image values that includes tag. This will allow us to use updatebot to automate updating images in the Helm chart values from pipelines and release scripts, i.e.

Given values.yaml:

```
activiti-cloud-audit:
  image: activiti/activiti-cloud-audit:7.1.320

activiti-cloud-query:
  image: activiti/activiti-cloud-query:7.1.122
```
When execute updatebot command:

```
updatebot push-version -k helm activiti/activiti-cloud-query 7.1.123 activiti/activiti-cloud-audit 7.1.321 
```
Then values.yaml will be updated in a single commit:

```
activiti-cloud-audit:
  image: activiti/activiti-cloud-audit:7.1.321

activiti-cloud-query:
  image: activiti/activiti-cloud-query:7.1.123
```